src/EditorFeatures/Core/LanguageServer/RazorInProcLanguageClient.cs

```diff
@@ -72,6 +72,7 @@ public override ServerCapabilities GetCapabilities(ClientCapabilities clientCapa
             {
                 vsServerCapabilities.SupportsDiagnosticRequests = true;
                 vsServerCapabilities.SpellCheckingProvider = true;
+                vsServerCapabilities.Experimental = SimplifyMethodHandler.SimplifyMethodMethodName;
```

Comment:
Consider making this a `Dictionary<string, bool>` or `Dictionary<string, object>` so we can add more things later. We do something similar [on the Razor side](https://github.com/dotnet/razor/blob/main/src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLanguageServerCapability.cs#L27)